projects/packages/connection/webpack.config.js

```diff
@@ -46,6 +53,12 @@ module.exports = [
 		plugins: [
 			...jetpackWebpackConfig.StandardPlugins( {
 				MiniCssExtractPlugin: { filename: '[name].css' },
+				DependencyExtractionPlugin: {
+					requestMap: {
+						// We don't want to externalize this package, we rather want to bundle it.
```

Comment:
Note this applies to all the bundles being built here, not just the jetpack-connection.js.

If you want to be safer, you'd want to do a separate config just for that one. That code might look something like this.
```js
const sharedConfig = {
    // Everything except .entry and .plugins
};

module.exports = [
    {
        ...sharedConfig,
        entry: {
            // All the existing entries except 'jetpack-connection'.
        },
        plugins: {
            ...jetpackWebpackConfig.StandardPlugins( {
                MiniCssExtractPlugin: { filename: '[name].css' },
            } ),
        }
    },
    {
        ...sharedConfig,
        entry: {
            // Only the 'jetpack-connection' entry.
        },
        plugins: {
            ...jetpackWebpackConfig.StandardPlugins( {
                MiniCssExtractPlugin: { filename: '[name].css' },
                DependencyExtractionPlugin: {
                    requestMap: {
                        // We don't want to externalize this package, we rather want to bundle it.
                        '@automattic/jetpack-connection': {},
                    },
                },
            } ),
        }
    },
];
```
You can see examples of this sort of thing in some of the other webpack configs around the monorepo.

Reply:
Done. Thank you.